docs/sphinx/reference-scenes.rst

```diff
@@ -554,10 +563,31 @@ Scene Item Functions
 
 ---------------------
 
+.. function:: void obs_sceneitem_set_transition(obs_sceneitem_t *item, bool show, obs_source_t *transition)
+
+   Sets a transition for showing or hiding a scene item.
+
+   :param item:       The target scene item
+   :param show:       If *true*, this will set the show transition; hide transition otherwise
```

Comment:
```suggestion
   :param show:       If *true*, this will set the show transition.
                      If *false*, this will set the hide transition.
```